test/unit/Heatmap/GMUGradientTest.swift

```diff
@@ -0,0 +1,141 @@
+/* Copyright (c) 2020 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import XCTest
+@testable import GoogleMapsUtils
+
+class GMUGradientTest: XCTestCase {
+  
+  var gradient          : GMUGradient!
```

Comment:
We can remove this as a property for this class and have it as a local variable instead.